src/index.js

```diff
@@ -182,4 +315,4 @@ class ReactToPrint extends React.Component {
 
 }
 
-export default ReactToPrint;
+export default ReactToPrint;
```

Comment:
Nit: it's generally a good idea to have a newline at the end of the file - players nicer with some parsing logic in some scripts.